src/components/Navbar/Navbar.js

```diff
@@ -5,6 +5,7 @@ import { GiHamburgerMenu } from "react-icons/gi";
 
 import { useStateContext } from "../../context/StateProvider";
 import { Link } from "react-router-dom";
+import { useAuthContext } from "../../context/AuthProvider";
```

Comment:
just a suggestion, you can use absolute paths while importing files. Checkout this article: https://dev.to/emanuelone/how-to-use-absolute-path-in-reactjs-503i

Reply:
Yup, i did this in other imports, missed this one.